File: PHP/UberX.php
<?php
require_once("Car.php");
class UberX extends Car
{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model)
    {
        parent::_construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    public function setPassenger($passenger)
    {
        if ($passenger == 4) {
            $this->passenger = $passenger;
        } else {
            echo "ocupamos 4 personas mas.";
        }
    }
}

File: Java/UberX.java
public class UberX extends Car {
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model) {
        super(license, driver);// en el video habian cambiado la propiedad del metodo constructor.
        this.brand = brand;
        this.model = model;

    }

    @Override
    void printDataCar() {

        super.printDataCar();
        System.out.println("Brand: " + brand + "Model: " + model);
    }
}
